### DISCUSSION TITLE
writing a blog post about dbt x window functions

### DISCUSSION THREAD

**Discussion by seanccsmith** [dbt-labs/docs.getdbt.com#2188] [Archive]
Good morning dbt! My team came up with a cool solution for using window functions in incremental dbt models, and we'd like to share it with the world. We spoke a little with Jeremy Cohen and he recommended we post here for some brainstorming discussion. Anyone from dbt think this is interesting?

**Comment by KiraFuruichi** [comment]
Hey @seanccsmith thanks for opening up this discussion! My name is Kira and I work as a technical writer for dbt Labs and work closely with folks for [Developer Blog](https://docs.getdbt.com/blog) contributions. 

For new Developer Blog posts, we're asking folks to instead open up a [Developer Blog Issue](https://github.com/dbt-labs/docs.getdbt.com/issues/new?assignees=&labels=content%2Cdeveloper+blog&template=contribute-to-developer-blog.yml) in this repo instead of discussions here (so sorry for the confusion). In that issue, we expect folks to have some sort of outline on what they'd like to write about, so we can triage it appropriately. You can read more about the contribution guide for Developer Blog material [here](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/developer-blog.md)!

Looking forward to reading more on window functions in incremental models! If you feel you're not quite ready for an outline yet for a blog post or want support in working on one, don't hesitate to take more time or we can chat here on ideas.
